packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -115,7 +115,7 @@ namespace FROSch {
         reduceAll(*this->MpiComm_,REDUCE_MAX,int(buildUniqueMap),ptr(&buildUniqueMapMax));
         
         if (buildUniqueMapMax>0) {
-            if (this->MpiComm_->getRank()==0) std::cout << "FROSch::CoarseSpace : WARNING: We re-build a unique map of the AssembledBasisMap_." << std::endl;
+            if (this->MpiComm_->getRank()==0) std::cout << "FROSch::CoarseSpace : WARNING: We re-build a unique map of AssembledBasisMap_." << std::endl;
```

Comment:
1. Warnings should go to `std:cerr`, not `std::cout`.
2. Warnings are best reserved for situations where the user did something wrong.  If the user didn't do anything wrong here, then the warning is just useless noise.  Users will report it as a bug and developers will have to waste time explaining that nothing is wrong.

Reply:
I just made another commit, which should take care of your three comments.